README.md

```diff
@@ -119,6 +119,20 @@ g =
 
 Provide these 3 functions to `Simple.Animation.Animated.ui`
 
+The hook is just a wrapper arround Elm-UI `Element's. A new wrapper
+must be created for each element that you would like to animate.
+Though because Elm-UI `Element`s can all be wrapped in either `el`,
+`row` or `column` that is usually all we need.
+
+To do that first we need to add a helper function so that wrappers can
+can be created more easily. Just import `elm-simple-animation` with
+```
+import Simple.Animation as Animation exposing (Animation)
+import Simple.Animation.Animated as Animated
```

Comment:
Maybe take the `as Animated` bit off so the code below will work

Reply:
Oops! Thanks!